Northstar.Client/mod.json

```diff
@@ -1,123 +1,123 @@
 {
-	"Name": "Northstar.Client",
-	"Description": "Various ui and client changes to fix bugs and add better support for mods",
-	"Version": "1.9.0",
-	"LoadPriority": 0,
-	"ConVars": [
-		{
-			"Name": "filter_hide_empty",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_hide_full",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_hide_protected",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_map",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_gamemode",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_mods",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "filter_map_hide_locked",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "modlist_show_convars",
-			"DefaultValue": "0",
-			"Flags": 16777216
-		},
-		{
-			"Name": "modlist_reverse",
-			"DefaultValue": "0",
-			"Flags": 16777216
-		}
-	],
-	"Scripts": [
-		{
-			"Path": "_custom_codecallbacks_client.gnut",
-			"RunOn": "CLIENT",
-			"ClientCallback": {
-				"Before": "NSSetupChathooksClient"
-			}
-		},
-		{
-			"Path": "client/cl_chat.gnut",
-			"RunOn": "CLIENT"
-		},
-		{
-			"Path": "ui/menu_ns_modmenu.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"Before": "AddNorthstarModMenu",
-				"After": "AddNorthstarModMenu_MainMenuFooter"
-			}
-		},
-		{
-			"Path": "ui/menu_ns_setversionlabel.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"After": "NS_SetVersionLabel"
-			}
-		},
-		{
-			"Path": "presence/ui_presence.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"After": "NSUpdateGameStateUIStart"
-			}
-		},
-		{
-			"Path": "presence/cl_presence.nut",
-			"RunOn": "CLIENT",
-			"ClientCallback": {
-				"After": "NSUpdateGameStateClientStart"
-			}
-		},
-		{
-			"Path": "ui/menu_ns_serverbrowser.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"Before": "AddNorthstarServerBrowserMenu"
-			}
-		},
-		{
-			"Path": "ui/menu_ns_connect_password.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"Before": "AddNorthstarConnectWithPasswordMenu"
-			}
-		},
-		{
-			"Path": "ui/menu_ns_custom_match_settings_categories.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"Before": "AddNorthstarCustomMatchSettingsCategoryMenu"
-			}
-		},
-		{
-			"Path": "ui/menu_ns_custom_match_settings.nut",
-			"RunOn": "UI",
-			"UICallback": {
-				"Before": "AddNorthstarCustomMatchSettingsMenu"
-			}
-		},
-		{
-			"Path": "ui/controller_prompts.nut",
-			"RunOn": "UI"
-		}
-	],
-	"Localisation": [
-		"resource/northstar_client_localisation_%language%.txt"
-	]
+  "Name": "Northstar.Client",
```

Comment:
`Northstar.Client/mod.json` and `Northstar.CustomServers/mod.json` both have indentation/formatting changes that I assume where caused by running some auto-formatting and should be reverted. ^^

Reply:
pain